application/src/main/java/org/opentripplanner/raptor/RaptorService.java

```diff
@@ -23,8 +25,16 @@ public class RaptorService<T extends RaptorTripSchedule> {
 
   private final RaptorConfig<T> config;
 
-  public RaptorService(RaptorConfig<T> config) {
+  @Nullable
+  private final ExtraMcRouterSearch<T> extraMcSearch;
+
+  public RaptorService(RaptorConfig<T> config, @Nullable ExtraMcRouterSearch<T> extraMcSearch) {
     this.config = config;
+    this.extraMcSearch = extraMcSearch;
+  }
```

Comment:
Maybe add a note that the extraMcSearch will be ignored for non-mc searches

Reply:
I have added doc to the ExtraMcRouterSearch class - what happens if the request is a standard search. This would apply to all other multi-criteria input parameters as well, I think it should be obvious that if you are running a standard search - all multi-criteria parameters are ignored.